package.json

```diff
@@ -75,6 +75,7 @@
     "@bbc/psammead-locales": "^4.0.0",
     "@bbc/psammead-media-indicator": "^2.6.19",
     "@bbc/psammead-media-player": "^2.3.7",
+    "@bbc/psammead-most-read": "0.4.0-alpha.1",
```

Comment:
Looking good! We can't merge this until the component is out of alpha though right?

Reply:
yup